nexus_constructor/model/component.py

```diff
@@ -303,6 +303,32 @@ def record_pixel_mapping(self, pixel_mapping: PixelMapping):
             "int64",
         )
 
+    def _create_transformation_vectors_for_pixel_offsets(self) -> List[QVector3D]:
+        """
+        Construct a transformation (as a QVector3D) for each pixel offset
+        """
+        x_offsets = self.get_field_value("x_pixel_offset")
```

Comment:
could this throw? in which case i think wrap all three in a ValueError try/except

Reply:
I've caught the exception in `get_field_value`